docs/index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Lab NASA 201</title>
+    <script src="intro_to_js/script.js"></script>
+</head>
+
+<!-- 
+    section: Agrupa contenido que esta relacionado. se usa cuando creamos bloques cuyo contenido es parte de un bloque total.
+
+    article: es un contenedor que representa contenido independiente, osea que tendria sentido por si mismo.  Es un componente de la pagina de contenido independiente porque podriamos leer uno sin haber leido el resto, y seguiria teniendo sentido.  esto implica que esta etiqueta puede tener un heater y un footer propios.
+
+    aside: se utiliza para representar contenido relaciopnado pero que no forma parte del contenido principal.
+
+    tambien existe el caso en el que un article contenga varias sections.
+
+    este ejemplo es un section que contiene varios articles
+-->
+<body>
+    <header>
+        <!-- Soy un elemento de bloquer -->
+        <h1>My First WEB Page</h1>
+        
+    </header>
+    <img src="images/oip.jpeg" alt="logo"> <br><br>
+    <p>
+        There are daily tasks in our homes that cannot wait and that we must attend to as soon as possible but we do not have time or we do not have those skills.
+        Don't worry about those things anymore, we can do it for you!
+    </p>
+     <!-- Soy un elemento de linea -->
+        <span>Plumbing</span> <br>
+        <span>Roofs</span> <br>
+        <span>Finishes</span> <br>
+        <span>Paint</span> <br>
+        <span>Carpentry</span> <br>
+        <span>locksmith</span> <br>
+        <span>welding</span> <br><br>
```

Comment:
FYI, a better element to use for this kind of list would be <ul> with one <li> element for each list item: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul